Base/res/js/Spreadsheet/runtime.js

```diff
@@ -403,7 +407,9 @@ randRange.__documentation = JSON.stringify({
     argc: 2,
     argnames: ["start", "end"],
     doc: "Returns a random number in the range (`start`, `end`)",
-    examples: {},
+    examples: {
+        'range("A1", "C4")': "Returns a random number within A1:C4"
```

Comment:
This is not how it works, `range` gives you...well, a range
randRange doesn't take cell names, it takes numbers (it's basically `Math.random`)

Reply:
You're correct, I'm not sure what I was looking at last night